splunk_connect_for_snmp/splunk/tasks.py

```diff
@@ -238,8 +239,19 @@ def prepare_trap_data(work):
             "host": work["address"],
             "index": SPLUNK_HEC_INDEX_EVENTS,
         }
-        events.append(json.dumps(event, indent=None))
-
+        if SPLUNK_CONGREGATE_TRAPS_EVENTS:
```

Comment:
I think I'd create a new function here for this if statement. Or I'm thinking - if we just do `events.append` for both, and run `json.dumps` at the end - won't it return the same result?

Reply:
I'll add new function. For events I think dumps can be done at the end if I add the loop.